knowledge_repo/app/templates/index-base.html

```diff
@@ -98,12 +82,51 @@
 <script type="text/javascript">
 
     $( document ).ready(function() {
-     $.ajaxSetup({
-      cache: true
-    });
+      $.ajaxSetup({
+        cache: true
+      });
+
+      function getParameters() {
```

Comment:
We know what it'll have and we already do the parsing. We could potentially pass it into the function here as a string too to make it easy to tack onto the buttons